custom_components/tuya_local/devices/zigbee_dv7vcs_human_presence_sensor.yaml

```diff
@@ -0,0 +1,115 @@
+name: human presence sensor
+products:
+  - id: sxm7l9xa
+    name: Human presence sensor
+primary_entity:
+  entity: binary_sensor
+  class: occupancy
+  dps:
+    - id: 105
+      type: string
+      name: sensor
+      mapping:
+        - dps_val: presence
+          value: true
+        - dps_val: none
+          value: false
+secondary_entities:
+  - entity: number
+    name: Sensitivity
+    category: config
+    icon: "mdi:motion-sensor"
+    dps:
+      - id: 106
+        type: integer
+        name: value
+        range:
+          min: 1
+          max: 10
+  - entity: number
+    name: Minimum distance
+    category: config
+    icon: "mdi:arrow-collapse-left"
+    dps:
+      - id: 108
+        type: integer
+        name: value
+        unit: m
+        range:
+          min: 0
+          max: 950
+        mapping:
+          - scale: 10
+            step: 10
+  - entity: number
+    name: Maximum distance
+    category: config
+    icon: "mdi:arrow-collapse-right"
+    dps:
+      - id: 107
+        type: integer
+        name: value
+        unit: m
+        range:
+          min: 0
+          max: 950
+        mapping:
+          - scale: 10
+            step: 10
+  - entity: sensor
+    name: Target distance
```

Comment:
I think the name is not needed, as the class: distance should have suitable translated names, and this is the only sensor of that class.